langchain/src/callbacks/manager.ts

```diff
@@ -395,8 +395,15 @@ export class CallbackManager
     inherit = true
   ): CallbackManager {
     const manager = new CallbackManager(this._parentRunId);
-    manager.setHandlers([...this.handlers].map((handler) => handler.copy()));
+    for (const handler of this.handlers) {
```

Comment:
@agola11 this wasn't right, it was losing the "inheritable" flag on each handler

Reply:
ah yeah, thanks for fixing.